c/dataservice.c

```diff
@@ -298,13 +305,17 @@ WebPlugin *makeWebPlugin(char *pluginLocation, JsonObject *pluginDefintion, Inte
       char *type = jsonObjectGetString(serviceDef, "type");
 
       if (!type || !strcmp(type, "service")) {
-        plugin->dataServices[k++] = makeDataService(plugin, jsonArrayGetObject(dataServices, i), NULL, internalAPIMap);
+        plugin->dataServices[k++] = makeDataService(plugin, jsonArrayGetObject(dataServices, i), NULL, internalAPIMap,
+                                                    idMultiplier, pluginLogLevel);
+        (*idMultiplier++);
```

Comment:
@jordanfilteau1995 , @1000TurquoisePogs  This has no effect - ```*``` and ```++``` have the same precedence but since they get processed right-to-left, pointer ```idMultiplier``` gets incremented first and then dereferenced.

Reply:
Neat. Thanks!